pkg/spec/config_linux.go

```diff
@@ -74,7 +74,7 @@ func devicesFromPath(g *generate.Generator, devicePath string) error {
 		return nil
 	}
 
-	return addDevice(g, devicePath)
+	return addDevice(g, strings.Join(append([]string{resolvedDevicePath}, devs[1:]...), ":"))
```

Comment:
Do we need a check that devs[1] exists here? We have one in the if above, looks like, but I don't think we ever validate if otherwise

Reply:
I tested this and it looks like it works.

devs[1:] with devs length of 1 works in my tests.

Reply:
Just doing podman run --device /dev/xdg1 ...

Would blow  up, which we have tests for.

Reply:
Alright, SGTM then